extracts/v_extracts.nj_position_control_roster.sql

```diff
@@ -0,0 +1,49 @@
+USE gabby
+GO
+
+CREATE OR ALTER VIEW extracts.nj_position_control_roster AS
+
+--CTE for only titles and entities needed (including all NJ staff and CEO and CFO from KTAF)--
+WITH nj_reportable AS (
```

Comment:
this could be turned into a WHERE statement on the main SELECT

i.e. WHERE job_title IN ... OR business_unit IN ...

Reply:
Got it, changed.